lib/_constants.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE_HIGHLIGHT: 'gmusic-theme-highlight',
```

Comment:
If we are using concatenation, then we should no longer need this highlighting class (since there's nothing to override via HTML elements).